libsigner/src/events.rs

```diff
@@ -258,7 +267,7 @@ impl EventReceiver for SignerEventReceiver {
     /// Errors are recoverable -- the caller should call this method again even if it returns an
     /// error.
     fn next_event(&mut self) -> Result<SignerEvent, EventError> {
-        self.with_server(|event_receiver, http_server, is_mainnet| {
+        self.with_server(|event_receiver, http_server, _is_mainnet| {
```

Comment:
This is a nit and doesn't need to be done here if you just want to merge, but we should change with_server fcn sig since it no longer needs is_mainnet